### PR TITLE
Add conditional probes

### PR DESCRIPTION
**Pull request by st-myk**
## Checklist

- [x] Lint and unit tests pass locally with my changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added documentation to describe my feature.
- [x] I have squashed my commits into logic units.
- [x] I have described the changes in the commit messages.

## Summary by CodeRabbit

- **New Features**
	- Updated version number to 0.5.12, indicating a new release that may include enhancements or bug fixes.
- **Documentation**
	- Revised version number in the README to reflect the new release.
- **Configuration Changes**
	- Adjusted deployment configurations for liveness and readiness probes for improved flexibility.
	- Added comments in the values file to clarify the optional nature of probe configurations.

### CODE REVIEW

File: charts/weblate/values.yaml [hunk 1]
```diff
@@ -175,11 +175,13 @@ tolerations: []
 
 affinity: {}
 
+# Can be removed when running Weblate serivice without probes configured
```
Comment (nijel) [inline charts/weblate/values.yaml:178]:
```suggestion
# Can be removed when running Weblate service without probes configured
```

File: charts/weblate/values.yaml [hunk 2]
```diff
@@ -175,11 +175,13 @@ tolerations: []
...
 livenessProbe:
   initialDelaySeconds: 300
   periodSeconds: 30
   failureThreshold: 10
 
+# Can be removed when running Weblate serivice without probes configured
```
Comment (nijel) [inline charts/weblate/values.yaml:184]:
```suggestion
# Can be removed when running Weblate service without probes configured
```

**Comment by notz** [comment]
@nijel Are you sure this is working as expected? We get the k8s default liveness probes and because of this containers are not starting after this changes (they are killed before startup finished -> no delay seconds). Helm liveness values seems not to be applied any more.

**Comment by nijel** [comment]
@notz Apparently it is not, but I didn't spot issue in it because I lack Kubernetes knowledge. It is now fixed via https://github.com/WeblateOrg/helm/pull/467 thanks to @Mart-Kuc.

This repository is looking for maintainers, see https://github.com/WeblateOrg/helm/issues/348

**Pull request merged**